interface/billing/print_daysheet_report_num1.php

```diff
@@ -162,20 +162,27 @@
 // this loop gathers the user and provider numbers
 foreach ($ret as $iter) {
   $catch_user[] = $iter{'user'};
-  $catch_provider[] = $iter{'provider_id'};
+  $catch_provider[] = $iter{'enc_provider_id'};
+  if($iter{'code_type'} == 'Patient Payment' && ($iter{'paytype'} == 'PCP' || $iter{'paytype'} == 'PP') )
+    $catch_payment_method[]  = $iter{'pay_method'};
```

Comment:
Copays?

Reply:
![image](https://user-images.githubusercontent.com/4655366/40510165-d0d462fe-5f69-11e8-905e-feabcfb66e61.png)
 they are a payment method

Reply:
Ah, not a "paytype".  Gotcha.